app/models/comment.rb

```diff
@@ -475,4 +475,13 @@ def render_body
     end
     body
   end
+
+  def self.find_by_tag_and_author(tagname, userid)
```

Comment:
Woohoo!!!! 🎉 

I wonder if we could write a simple unit test for this great method??? Amazing work!

Reply:
Yes, sure!